### DISCUSSION TITLE
Setting baseUrl in cypress.json does nothing...

### DISCUSSION THREAD

**Discussion by Mugane** [cypress-io/cypress#16178] [Questions and Help] [answered]
![image](https://user-images.githubusercontent.com/13005927/115922284-b3711580-a44a-11eb-9ef6-d7aa6260777e.png)

Well, it looks like it's setting the baseUrl config variable but when `cy.visit()` is called it gives this error:
```
cy.visit() must be called with a url or an options object containing a url as its 1st argumentLearn more
```
Yet it clearly states in the documentation that baseUrl can be set in cypress.json. So... How does one set the default value for baseUrl? How does one overwrite it at runtime?

From [Cypress.io](https://docs.cypress.io/guides/guides/environment-variables#Setting)
> baseUrl can be set in your configuration file (cypress.json by default) - and then you can set an environment variable in your OS to override it like shown below.
> ...
Here's the cypress.json file: 
```
{
	"baseUrl": "http://localhost:3001/#/",
	"ignoreTestFiles": "**/examples/*.spec.js",
	"env": {
		"API_URL": "http://localhost:4444/",
	}
}

**Comment by bahmutov** [comment, marked as answer]
Have you tried `cy.visit('/')`? This should visit the `baseUrl` specified in the `cypress.json` file

https://docs.cypress.io/api/commands/visit#Usage

**Reply by Mugane** [reply]
Ok this does actually work, as does `cy.visit('')` - thanks. However, `cy.visit()` does *not* work.

**Reply by iSuslov** [reply]
> Have you tried `cy.visit('/')`? This should visit the `baseUrl` specified in the `cypress.json` file
> 
> https://docs.cypress.io/api/commands/visit#Usage

You should mention it in your documentation. Your documentation for env variables now says:
`cy.visit() and cy.request() are automatically prefixed with this value - avoiding the need to specify them.`
which is not clear and leads to such outcome. 

https://docs.cypress.io/guides/guides/environment-variables

**Reply by mycargus** [reply]
I agree the documentation is confusing as written. Clarification would be great. :)  

Thanks for a great product, I'm enjoying Cypress!

**Reply by emilyrohrbough** [reply]
@iSuslov Could you open an issue to https://github.com/cypress-io/cypress-documentation/ about this confusing text around the baseURL option?

**Reply by Mugane** [reply]
> @iSuslov Could you open an issue to https://github.com/cypress-io/cypress-documentation/ about this confusing text around the baseURL option?

I'm confused, as a maintainer wouldn't addressing an issue like this basically be the purpose of your maintenance role? Why do you need a third party to submit a ticket or create a report or whatever? All the information is already provided here.
